packages/protocol/contracts/L1/v1/V1Proposing.sol

```diff
@@ -19,23 +19,30 @@ library V1Proposing {
     using SafeCastUpgradeable for uint256;
     using LibData for LibData.State;
 
-    event BlockCommitted(bytes32 hash, uint256 validSince);
+    event BlockCommitted(
+        uint64 commitSlot,
+        uint64 commitHeight,
+        bytes32 commitHash
+    );
     event BlockProposed(uint256 indexed id, LibData.BlockMetadata meta);
 
-    function commitBlock(LibData.State storage s, bytes32 commitHash) public {
+    function commitBlock(
+        LibData.State storage s,
+        uint64 commitSlot,
+        bytes32 commitHash
+    ) public {
+        require(LibConstants.K_COMMIT_DELAY_CONFIRMATIONS > 0, "L1:disabled");
```

Comment:
Using `assert` for checking against constants would be the more standard way, but doesn't really matter.

Reply:
Switched to use asserts where it's appropriate.